fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryUnorderedDistinctPlan.java

```diff
@@ -159,4 +161,12 @@ public void logPlanStructure(StoreTimer timer) {
     public int getComplexity() {
         return 1 + getInner().getComplexity();
     }
+
+    @Nonnull
+    @Override
+    public PlannerGraph rewritePlannerGraph(@Nonnull final List<? extends PlannerGraph> childGraphs) {
+        return PlannerGraph.fromNodeAndChildGraphs(
+                new PlannerGraph.OperatorNodeWithInfo(this, NodeInfo.UNORDERED_DISTINCT_OPERATOR),
```

Comment:
I think that this operator needs to include the `comparisonKey`, which is what's used to determine distinctness.

For now, we can just use `KeyExpression.toString()` for that.

Reply:
good catch. done.